docs/MockFunctionAPI.md

```diff
@@ -316,3 +316,133 @@ test('async test', async () => {
   await asyncMock(); // throws "Async error"
 });
 ```
+
+## TypeScript
+
+Jest itself is written in [TypeScript](https://www.typescriptlang.org).
+
+If you are using [Create React App](https://create-react-app.dev) then the [TypeScript template](https://create-react-app.dev/docs/adding-typescript/) has everything you need to start writing tests in TypeScript.
+
+Otherwise, please see our [Getting Started](https://jestjs.io/docs/en/getting-started#using-typescript) guide for to get setup with TypeScript.
```

Comment:
```suggestion
Otherwise, please see our [Getting Started](GettingStarted.md#using-typescript) guide for to get setup with TypeScript.
```

or something like that - same comment applies to the other links